### PR TITLE
[CWS] Disable unused registry event types

### PR DESCRIPTION
**Pull request by YoannGh**
### What does this PR do?

Reconfigure ETW to only get registry events for which we have agent rules to match against.

### CODE REVIEW

File: pkg/security/probe/probe_windows.go [hunk 1]
```diff
@@ -444,11 +457,9 @@ func (p *WindowsProbe) auditEtw(ecb etwCallback) error {
 		case etw.DDGUID(p.auditguid):
 			switch e.EventHeader.EventDescriptor.ID {
 			case idObjectPermsChange:
-				if p.isChangePermissionEnabled {
```
Comment (paulcacheux) [inline pkg/security/probe/probe_windows.go:447]: Oups, thanks !

File: pkg/security/probe/probe_windows.go [hunk 2]
```diff
@@ -444,11 +457,9 @@ func (p *WindowsProbe) auditEtw(ecb etwCallback) error {
...
-					if pc, err := p.parseObjectPermsChange(e); err == nil {
-						log.Infof("Received objectPermsChange event %d %s\n", e.EventHeader.EventDescriptor.ID, pc)
-						ecb(pc, e.EventHeader.ProcessID)
-					}
+				if pc, err := p.parseObjectPermsChange(e); err == nil {
+					log.Infof("Received objectPermsChange event %d %s\n", e.EventHeader.EventDescriptor.ID, pc)
```
Comment (paulcacheux) [inline pkg/security/probe/probe_windows.go:461]: Can we make this a trace log, AFAIK all other logs like that are trace logs

**Review by paulcacheux — APPROVED** [review]
very nice !

File: pkg/security/probe/probe_windows.go (hunk 2 above)
Comment (YoannGh) [reply pkg/security/probe/probe_windows.go]: Done

Comment (paulcacheux) [reply pkg/security/probe/probe_windows.go]: thanks !

**Pull request merged**